packages/as-transformer/src/transformers/massaExportImports.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import {
+  IdentifierExpression,
+  ImportDeclaration,
+  ImportStatement,
+} from 'assemblyscript/dist/assemblyscript.js';
+import { Update, GlobalUpdates } from './interfaces/Update.js';
+// import { IExpressionTransformer } from './interfaces/IExpressionTransformer';
+
+/**
+ * The MassaExportCalls is a specific transformer that works by replacing a any massa exported function calls
+ * by their transformed version.
+ */
```

Comment:
MassaExportCalls should be MassaExportImports

Does this explaination fit more the purpose of this class ?

```
/**
 * MassaExportImports class specializes in transforming AssemblyScript import statements.
 * 
 * It replaces calls to any function exported with the "MassaExport" keyword with an underscored version
 * of the same function. This class works alongside an AST transformation process and is used primarily
 * to update exported functions references in import declarations.
 */
 ```
 
 If not ignore it

Reply:
close enough, I'll update it thanks.

Reply:
On second thoughts, i'll do the documentation of this class in the appropriate issue, since this class is not used at the moment. If it's ok ofc.

Reply:
No problem